src/main/java/frc/robot/VisionWindow.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot;
+
+import java.awt.Graphics;
+
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.Timer;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class VisionWindow extends JPanel implements ActionListener {
```

Comment:
What's the purpose of this class?

Reply:
this is for viewing the pose from the network tables, more info in the comments in that class in the new commit